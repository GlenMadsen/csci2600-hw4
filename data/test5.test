r
111
117
r
112
91
r
34
113
r
1002
1002
r
1324
128
r
91
51
q