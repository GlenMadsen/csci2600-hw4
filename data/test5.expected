First building id/name, followed by Enter: 
Second building id/name, followed by Enter: 
Unknown building: [111]
Unknown building: [117]
First building id/name, followed by Enter: 
Second building id/name, followed by Enter: 
Unknown building: [112]
First building id/name, followed by Enter: 
Second building id/name, followed by Enter: 
Unknown building: [113]
First building id/name, followed by Enter: 
Second building id/name, followed by Enter: 
Unknown building: [1002]
First building id/name, followed by Enter: 
Second building id/name, followed by Enter: 
Unknown building: [1324]
Unknown building: [128]
First building id/name, followed by Enter: 
Second building id/name, followed by Enter: 
Path from OGE to 2144 Burdett Avenue:
	Walk NorthWest to (Intersection 112)
	Walk East to (Intersection 117)
	Walk East to (Intersection 111)
	Walk East to (Intersection 113)
	Walk SouthEast to (2144 Burdett Avenue)
Total distance: 539.140 pixel units.
